src/Microsoft.AspNet.OData.Shared/Builder/ODataConventionModelBuilder.cs

```diff
@@ -86,37 +87,29 @@ public class ODataConventionModelBuilder : ODataModelBuilder
         private Lazy<IDictionary<Type, List<Type>>> _allTypesWithDerivedTypeMapping;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="ODataConventionModelBuilder"/> class.
-        /// </summary>
-        public ODataConventionModelBuilder()
-        {
-            Initialize(new DefaultAssembliesResolver(), isQueryCompositionMode: false);
-        }
-
-        /// <summary>
         /// Initializes a new <see cref="ODataConventionModelBuilder"/>.
         /// </summary>
-        /// <param name="configuration">The <see cref="HttpConfiguration"/> to use.</param>
-        public ODataConventionModelBuilder(HttpConfiguration configuration)
-            : this(configuration, isQueryCompositionMode: false)
+        /// <param name="resolver">The <see cref="IWebApiAssembliesResolver"/> to use.</param>
+        internal ODataConventionModelBuilder(IWebApiAssembliesResolver resolver)
+            : this(resolver, isQueryCompositionMode: false)
```

Comment:
breaking? how will public call ctor?

Reply:
Step4 adds it back on the AspNet side, hence the partial tag being added here.